pyproject.toml

```diff
@@ -9,6 +9,9 @@ python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.25.1"
+pytest = "^6.2.4"
+blinker = "^1.4"
+selenium = "^3.141.0"
```

Comment:
Not a major concern, but worth noting that you could install these under the dev-dependencies section below (or using the `-D` flag when adding a new dependency with `poetry add`) so that production installations don't need to worry about these dependencies